soc/nordic/common/Kconfig

```
@@ -22,6 +22,12 @@ config MRAM_LATENCY_SYNC_TIMEOUT
 	help
 	  Timeout is given in milliseconds.
 
+config MRAM_LATENCY_AUTO_REQ
+	bool "Request MRAM without latency at start"
+	help
+	  When enabled then MRAM configuration without latency is requested
+	  during the initialization.
```

I think this requires clarification if the latency is applied indefinitely, or if there is any action to release this default request.

```suggestion
	  When enabled then MRAM configuration without latency is requested
	  during the initialization and is kept enabled until the mram_latency API user
	  calls mram_no_latency_sync_release().
```